packages/shared/lib/db/migrations/20240326141028_hash-key.cjs

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const utils = require('node:util');
+const crypto = require('node:crypto');
+
+const pbkdf2 = utils.promisify(crypto.pbkdf2);
+const ENCRYPTION_KEY = process.env['NANGO_ENCRYPTION_KEY'];
+
+function decrypt(enc, iv, authTag) {
+    const decipher = crypto.createDecipheriv('aes-256-gcm', Buffer.from(ENCRYPTION_KEY, 'base64'), Buffer.from(iv, 'base64'));
+    decipher.setAuthTag(Buffer.from(authTag, 'base64'));
+    let str = decipher.update(enc, 'base64', 'utf8');
+    str += decipher.final('utf8');
+    return str;
+}
```

Comment:
Because those file are .cjs we can't really import our own code. It's either work locally but not in test, or the opposite :/